chardet/sbcharsetprober.py

```diff
@@ -118,4 +118,7 @@ def get_confidence(self):
             r = r * self._freq_char / self._total_char
             if r >= 1.0:
                 r = 0.99
+	    #print 'Charset name: %s confidence = %f' % (self._model['charset_name'], r)
```

Comment:
Please either remove these lines entirely or change them to use `self.logger.debug`, like:

```python
self.logger.debug('Charset name: %s confidence = %f', self._model['charset_name'], r)
```